semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/NodeTypeStarters.scala

```diff
@@ -75,6 +75,10 @@ class NodeTypeStarters(cpg: Cpg) {
   def continue: Traversal[ControlStructure] =
     controlStructure.isContinue
 
+  @Doc("All throws (`ControlStructure` nodes)")
+  def throws: Traversal[ControlStructure] =
```

Comment:
It would've been better to name this `throw` for consistency with other starters, but this doesn't work since `throw` is a reserved word.

Reply:
makes sense. you could add an escaped alias which, but that's not everyones favorite, understandably
```
def `throw`: Traversa[ControlStructure] = throws
```